### ISSUE TITLE
MacOS Catalina freezes

### ISSUE THREAD

**Issue by jfrverdasca** [foxlet/macOS-Simple-KVM#499]
I'm facing a MacOS Catalina (10.15.7) freeze after some time without user interaction activity.
I disabled the screen saver (set time to "never") and because it is a virtual machine I disabled also the computer and screen pause (set it to "never") and unchecked the hard disk pause option. Now it works file and never freezes again!

**Comment by tombombadilom** [comment]
Same issue for me using big sur , there are nothing in the mac os x logs when the machine freeze. Big mistery

**Comment by tombombadilom** [comment]
my bad I just had to stop hard drive suspend to solve the problem 
https://support.apple.com/en-en/guide/mac-help/mchlp1168/mac

**Comment by curiousercreative** [comment]
can this be closed?

**Comment by jfrverdasca** [comment]
> can this be closed?

Yes. This freeze happens because the disk suspension. The solution is disable disk suspension in power options.
